html/modules/custom/reliefweb_fields/src/Plugin/Field/FieldWidget/ReliefWebOptions.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Drupal\reliefweb_fields\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\Plugin\Field\FieldWidget\OptionsButtonsWidget;
+use Drupal\Core\Field\Plugin\Field\FieldWidget\OptionsWidgetBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\reliefweb_utility\Traits\EntityDatabaseInfoTrait;
+
+/**
+ * Plugin implementation of the 'reliefweb_options' widget.
+ *
+ * @FieldWidget(
+ *   id = "reliefweb_options",
+ *   module = "reliefweb_fields",
+ *   label = @Translation("ReliefWeb Options with description"),
+ *   multiple_values = true,
+ *   field_types = {
+ *     "list_integer",
+ *     "list_string",
+ *     "list_float",
+ *     "boolean",
```

Comment:
The code seems to only work for entity reference field types so I think that should be removed.

Reply:
True